packages/core/src/specs/LottieAnimationViewNativeComponent.ts

```diff
@@ -27,9 +42,11 @@ export interface NativeProps extends ViewProps {
   enableMergePathsAndroidForKitKatAndAbove?: boolean;
   hardwareAccelerationAndroid?: boolean;
   cacheComposition?: boolean;
-  colorFilters?: ReadonlyArray<string>;
-  textFiltersAndroid?: ReadonlyArray<string>;
-  textFiltersIOS?: ReadonlyArray<string>;
+  colorFilters?: ReadonlyArray<ColorFilterStruct>;
+  // dummy that solves codegen issue when there's a ReadonlyArray<Object> without another Object prop
+  dummy?: Readonly<{ dummy: boolean }>;
```

Comment:
Is there an existing issue for that problem? That seems like a very strange way to solve this codegen problem.

Reply:
I'm creating it to add more visibility. I had a conversation about it with cipolleschi and Titozzz on discord. https://github.com/reactwg/react-native-new-architecture/discussions/104

Reply:
Is this solved? I am still dealing with it in react native 0.71-rc5. Can you verify?

Reply:
It hasn't been solved. I'll link the discussion here.